project-p3-forms-master/code/style.css

```diff
@@ -0,0 +1,247 @@
+/* 
+If you can see a peach background, the css file is correctly linked. 
+Feel free to change or remove this styling! 
+We want you focus A LOT on the styling for this project. 
+*/
+
+body {
+  background-color: #7e8a97;
+  font-family:'Handlee', cursive;
+ }
+
+* {
+  box-sizing: border-box;
+}
+
+form {
+ display: flex; 
+ text-align: center;
+ align-items: center;
+ justify-content: center;
+ flex-direction: column;
+ background: #e7dec8; 
+ border: 2px solid #30475e;
+ border-radius: 4px; 
+ max-width: 500px; 
+ padding: 20px; 
+ margin: 0 auto; 
+}
+
+h3 {
+  margin-top: 0;
+}
+  
+h2 {
+ font-size: 25px;
+ margin-top: 0; 
+ margin-bottom: 0; 
+}
+
+fieldset {
+  border: none;
+  background: #e7dec8; 
+  border-radius: 4px; 
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin: 10px; 
+  padding: 10px 0px 10px 10px;
+  box-shadow: 0 3px 5px 3px rgba(50, 50, 93, 0.15), 0 6px 8px 3px rgba(112, 157, 199, 0.15); 
+}
+
+label {
+  padding: 5px; 
+  text-align: left;
+  width: 300px;
+}
+
+input {
+ margin-top: 5px;
+ width: 350px; 
```

Comment:
The issue is rather that you have some elements that are too wide for mobile screens. If you add media queries, e.g.

```
@media (max-width: 668px) {
  fieldset {
    max-width: 350px;
  }

  input, select, .wrapper-row {
    max-width: 275px;
  }
}
```

it will work well on mobile :)

Reply:
Thank you! :)